src/python/grpcio/grpc/compat.py

```diff
@@ -0,0 +1,6 @@
+import six
+
+if six.PY2:
+  import cStringIO as StringIO
```

Comment:
Ah, good point! I guess since the code is already written against a module StringIO, py3 should `import io as StringIO`.